### PR TITLE
rc git-tools: Allow jumping between hunks

### LINKED ISSUE #2423: Implement `git` commands to jump to the previous/next modification in the buffer
Hi,

It'd be nice to have commands that jump from one hunk to the other, whose lines can already be highlighted with the `:git show-diff` command.

However I think it'd also be better if users didn't have to use `show-diff` in order to be able to jump to a hunk (to make use of the flag option that stores line coordinates), so some kind of refactoring would potentially have to take place.

HTH.

### PR DESCRIPTION
**Pull request by lenormf**
Fixes #2423

### CODE REVIEW

**Comment by lenormf** [comment]
Thanks, rebased on eba4002.

**Comment by lenormf** [comment]
Bump!

File: rc/extra/git-tools.kak [hunk 1]
```diff
@@ -146,6 +146,79 @@ Available commands:\n  add\n  rm\n  blame\n  commit\n  checkout\n  diff\n  hide-
         '
     }
 
+    jump_next_hunk() {
```
Comment (mawww) [inline rc/extra/git-tools.kak]: Thats around 80 new lines for next/prev, I would be more confortable with less, I think there should be a bit of room for simplification, and hopefully sharing code between the two implementation.

**Comment by lenormf** [comment]
The two functions don't have the same logic even though they both use a loop, I don't think factorizing for the sake of saving on the few lines that could be factorized is really beneficial.

Rebased on master.

**Pull request closed without merging**